release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -291,6 +291,32 @@ Currently, it is not supported to use Active Directory Federation Services with
 
 For the full list of OIDC providers, see xref:../authentication/identity_providers/configuring-oidc-identity-provider.adoc#identity-provider-oidc-supported_configuring-oidc-identity-provider[Supported OIDC providers].
 
+[id="ocp-4-11-auth-pod-security-admission"]
+==== Pod security admission
+
+link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[Pod security admission] is now enabled on {product-title}. 
+
+Pod admission is enforced by both pod security and security context constraint (SCC) admissions. Pod security admission runs globally with "privileged" enforcement and "restricted" audit logging and API warnings.
+
+A controller monitors the xref:../authentication/managing-security-context-constraints.adoc[SCC-related permissions] of service accounts in user-created namespaces and automatically labels these namespaces with link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security admission] `warn` and `audit` labels.
+
+To improve workload security in accordance with the "restricted" pod security profile, this release introduces SCCs that enforce pod security in accordance with new pod security admission controls. These SCCs are:
+
+* `restricted-v2`
+* `hostnetwork-v2`
+* `nonroot-v2`
+
+They correspond to older, similarly named SCCs, but with the following enhancements:
+
+* `ALL` capabilities are dropped from containers.
+* The `NET_BIND_SERVICE` capability can be added explicitly.
+* `seccompProfile` is set to `runtime/default` by default.
+* `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. 
```

Comment:
IMO we need a more detailed DELTA between v1 and v2. On top of that. we might want to be more specific here it allows those that develop solutions for OCP and cluster admins have better info to check if/when/how they are or are not impacted. See:

**Suggestions:**

Ensure that we clarify that: 

- a) we need explicitly to let the users know what was dropped before and now is not. That is important for those that need to evaluate/troubleshoot if a solution running on the cluster might or not be impacted. (i.e the workload was using capabilities that were NOT dropped before but will not which will not allow the workload to do what it must. )) 

- b) Make clear the requirements to be qualified as v2 which were not before with v1 for each case. 


```suggestion
* `ALL` capabilities are dropped from containers. Note that for the previous SCC policies for nonroot, restricted and hostnetwork ONLY `KILL`,`MKNOD`,`SETUID`,`SETGID` capabilities are dropped while the (v2) policies drops ALL. _(The new policies require you to either leave the dropped capabilities empty, or set it to ALL, or either allow a subset explicitly. )_
* The `NET_BIND_SERVICE` capability can be added explicitly.
* `seccompProfile` is set to `runtime/default` by default. V2 SCC policies require you to either leave seccompProfile empty or set it to runtime/default. _(Note that for the previous SCC policies you MUST leave empty.)_
* `allowPrivilegeEscalation` must be unset or set to `false`. 
```

c/c @stlaz

Reply:
@stlaz Any thoughts about this?

Reply:
Let's stay as brief as possible. I think a reasonable compromise is:

```suggestion
* `ALL` capabilities are dropped from containers. Previously only a small set of capabilities would have been dropped.
* The `NET_BIND_SERVICE` capability can be added explicitly.
* `seccompProfile` gets defaulted to `runtime/default` if unset. In previous releases, this field was required to stay empty.
* `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. Previously it was also allowed to be set to `true`.
```

Reply:
@maxwelldb as per discussion with @camilamacedo86, let's include the list of previously dropped caps:
```
`ALL` capabilities are dropped from containers. Previously only `KILL`, `MKNOD`, `SETUID`, `SETGID` were being dropped.
```

Reply:
```suggestion
* `ALL` capabilities are dropped from containers. Previously, only a small set of capabilities were dropped.
* The `NET_BIND_SERVICE` capability can now be added explicitly.
* `seccompProfile` is defaulted to `runtime/default` if unset. In previous releases, this field had to be empty.
* `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. Previously, a `true` value was allowed.
```
@stlaz